src/main/resources/templates/records.html

```diff
@@ -7,9 +7,7 @@
 <main id="main" role="main">
     <div th:replace="main.html::phase"></div>
     <div class="grid-row">
-        <div class="column-two-thirds">
-            <h1 th:text="${entries.size()} + ' records'"></h1>
```

Comment:
Yes `h1` is replaced by text i.e. `1-100 of 1000 records` or `56 records`. Have a look at the attached png's with the PR.